internal/cmd/schema-registry/command_cluster_describe.go

```diff
@@ -68,10 +70,20 @@ func (c *command) clusterDescribe(cmd *cobra.Command, _ []string) error {
 	if err != nil {
 		return err
 	}
-	cluster, err := c.Context.FetchSchemaRegistryByEnvironmentId(ctx, environmentId)
+	clusters, err := c.V2Client.GetSchemaRegistryClustersByEnvironment(environmentId)
 	if err != nil {
 		return err
 	}
+
+	cluster := clusters[0]
```

Comment:
Should we check the length to prevent a panic?

Reply:
length is being checked within the package before returning the list

Reply:
What do you think about moving that logic out here? It would be more explicit

Reply:
ok!